### PR TITLE
Add script for remove ledger

### PR DESCRIPTION
**Pull request by adenishchenko**
Signed-off-by: anton.denishchenko <anton.denishchenko@evernym.com>

### CODE REVIEW

File: scripts/remove_ledger.py
Comment (askolesov) [inline scripts/remove_ledger.py:13]: Let's rename `file_parth` to `ledger_name`. What do you think? It's not a path actually.

File: scripts/remove_ledger.py
Comment (askolesov) [inline scripts/remove_ledger.py:20]:
```suggestion
        print('Can`t delete built in ledger: ' + file_path)
```

File: scripts/remove_ledger.py
Comment (askolesov) [inline scripts/remove_ledger.py:18]:
```suggestion
            print('The follow directory was deleted: ' + path.name)
```

**Review by askolesov — CHANGES_REQUESTED** [review]
Looks good!

**Comment by askolesov** [comment]
(ci) test this please

File: scripts/remove_ledger.py
Comment (WadeBarnes) [inline scripts/remove_ledger.py:11]: Should `domen` be `domain`?

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
I'd highly recommend the script provide some feedback to the user and wait for confirmation. 
 It should clearly indicate it's about to delete a ledger, which ledger it's about to delete, describe the consequences, and then wait for confirmation from the user.

Some help text for the script would also be useful for those that are curious about what the script does, but don't want to read and fully understand the code.

**Comment by askolesov** [comment]
@WadeBarnes Great ideas

File: scripts/remove_ledger.py
Comment (WadeBarnes) [inline scripts/remove_ledger.py:11]:
```suggestion
    print('The follow directories will be deleted:')
```

File: scripts/remove_ledger.py
Comment (WadeBarnes) [inline scripts/remove_ledger.py:16]:
```suggestion
    print('Deleting a ledger is an irrevocable operation.\nProceed only if you know the consequences.')
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Looking good.  A few minor suggestions.

File: scripts/remove_ledger.py
Comment (askolesov) [inline scripts/remove_ledger.py:11]:
```suggestion
    print('The following directories will be deleted:')
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
DCO sign-off is missing from your last commit.

**Comment by WadeBarnes** [comment]
(ic) test this please.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Code is failing static analysis; https://build.sovrin.org/job/indy-node/job/indy-node-verify-x86_64/2836/execution/node/18/log/

**Review by askolesov — APPROVED** [review]

**Comment by askolesov** [comment]
(ci) test this please

**Comment by askolesov** [comment]
(ci) ok to test

**Comment by askolesov** [comment]
@WadeBarnes It seems that all comments were processed. Could you review this again, please?

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
@Toktar, @skhoroshavin, We just need approval from a code owner.

**Review by Toktar — APPROVED** [review]

**Pull request merged**
